build.assets/version-ops/cmd/check/main_test.go

```diff
@@ -121,6 +132,11 @@ type fakeGitHub struct {
 	releases []string
 }
 
-func (f *fakeGitHub) ListReleases(ctx context.Context, organization string, repository string) ([]string, error) {
-	return f.releases, nil
+func (f *fakeGitHub) ListReleases(ctx context.Context, organization, repository string) ([]github.RepositoryRelease, error) {
+	ghReleases := make([]github.RepositoryRelease, 0)
+	for _, r := range f.releases {
+		tag := r
```

Comment:
There's no need for the temporary `tag` variable here, is there?

`r` will be a copy of the string on each iteration, so `TagName: &r` seems fine to me.